src/Symfony/Bridge/Twig/Extension/EmojiExtension.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Extension;
+
+use Symfony\Component\Emoji\EmojiTransliterator;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+/**
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ */
+final class EmojiExtension extends AbstractExtension
+{
+    private static array $transliterators = [];
+
+    public function __construct()
+    {
+        if (!class_exists(EmojiTransliterator::class)) {
+            throw new \LogicException('You cannot use the "emojify" filter as the "Emoji" component is not installed. Try running "composer require symfony/emoji".');
+        }
+    }
+
+    public function getFilters(): array
+    {
+        return [
+            new TwigFilter('emojify', $this->emojify(...)),
+        ];
+    }
+
+    public function emojify(string $string, string $catalog = 'slack'): string
```

Comment:
Why `slack` is the default catalogue ? (could be github or other provider)

Reply:
I think, the default should be a constructor parameter. If I wanted to use let's say the GitHub catalog, I don't see myself passing `github` to the filter in every single Twig template I use the filter in. I would want to set this once for the entire project.

Reply:
> Why `slack` is the default catalogue ? (could be github or other provider)
 
One must be choosen. And I use slack in my app 🙈 
And this catalog looks more complete than github one.

But I really don't mind to put something else.